nomad/grapl-generator.nomad

```diff
@@ -0,0 +1,72 @@
+variable "tenant_id" {
+    type = string
+    description = "The opaque tenant id."
+}
+
+variable "generator_ingest" {
+    type = string
+    description = "The expected data ingest type. e.g. sysmon."
```

Comment:
I think this should really be a UUID, not a string. See https://github.com/grapl-security/grapl-rfcs/pull/1#discussion_r678683586 for rationale.

Reply:
We can update the comment specifically if we want, but the `type` field is a type in HCL2 (https://www.nomadproject.io/docs/job-specification/hcl2/expressions#types-and-values).